### ISSUE TITLE
Question: how to check undo stack size?

### ISSUE THREAD

**Issue by obaid** [StephenHaney/redux-time-travel#4]
Is there a way to know if there are any undo's left in the stack?

**Comment by StephenHaney** [comment]
Hey @obaid, not currently... but it's safe to submit `TIME_TRAVEL_BACKWARD` actions even when there's nothing left in the history. Nothing will happen at all.

What's your use case and how would you prefer to get your current place in the stack? Maybe we can extend it to make the info available.

**Comment by obaid** [comment]
I am using it an a simple text editor. I noticed that there are no side-effects when submitting `TIME_TRAVEL_BACKWARDS` action but for a good UX it would be good to know if the undo stack doesn't have any changes left.

I see that there is a `currentHistoryIndex` that tracks which index are we on. Can we somehow expose it to the public API? (I am not really that well versed in redux to suggest how)

**Comment by StephenHaney** [comment]
Yes, one of the things I'd like to do is add an option to store all of the time travel info in the Redux store itself, including data like the currentHistoryIndex.

Also, I've been meaning to bump maxHistoryLength. It's way too conservative by default. Unless your diffs are huge, you can set the historyLength to hundreds or thousands of history events without performance concerns.

Hope it helps! Thanks for the feedback - I'll prioritize an option to expose that data.
